kustomize/components/spanner/kustomization.yaml

```diff
@@ -14,9 +14,6 @@
 
 apiVersion: kustomize.config.k8s.io/v1alpha1
 kind: Component
-images:
-- name: gcr.io/google-samples/microservices-demo/cartservice
-  newTag: v0.4.0-spanner
```

Comment:
**Rationale for change:**
* The Cloud Spanner deployment variation was added when the latest release of Online Boutique was 0.4.0.
* This means that the `cartservice:v0.4.0` image doesn't contain the code necessary to handle a Cloud Spanner databases.
* But the `cartservice:v0.4.1` does! (i.e., the `-spanner` suffix is no longer necessary after v0.4.1 [inclusive])
* For users that wanted to use this Cloud Spanner variation, we built an additional image, `cartservice:v0.4.1-spanner`.
* Online Boutique 0.4.1 was released today.